tf2onnx/tf_loader.py

```diff
@@ -684,8 +684,12 @@ def tf_optimize_grappler(input_names, output_names, graph_def, fold_constant=Non
     # depends on so for now don't turn this on, fold_constant is always enabled now.
     rewrite_options.optimizers[:] = [
         # 'pruning', 'constfold', 'arithmetic', 'dependency', 'function',
-        'constfold', 'function'
+        'function', 'dependency'
     ]
+    if is_tf2():
```

Comment:
Looks like this flag was effective only after v2.4.0, so this is not good enough and needs to be changed like below:

if LooseVersion(tf.__version__) > "2.4":